src/explore-education-statistics-admin/src/components/editable/EditablePageModeToggle.module.scss

```diff
@@ -1,11 +1,12 @@
 @import '~govuk-frontend/govuk/base';
-
+/* stylelint-disable max-nesting-depth */
```

Comment:
Shouldn't need to nest as we can hoist things like `open` up to the top level. As we're in a CSS module, there aren't any potential scoping issues.

Reply:
It's a bit awkward because it needs some specificity to override the govuk styles, but I've rejigged it so the linter is happy.